src/main/java/com/cgm/infolab/controller/UserApiController.java

```diff
@@ -25,10 +28,26 @@ public UserApiController(UserRepository userRepository) {
     }
 
     @GetMapping("/api/users")
-    public List<UserDto> getUsername(@RequestParam("user") String user) {
+    public List<UserDto> getUsername(@RequestParam("user") String user,
+                                     @RequestParam(required = false, name = "page[size]") Integer pageSize,
```

Comment:
`"page[size]"`, `"page[before]"` e `"page[after]"` le userai praticamente in tutte le api. Ci farei delle constanti. Potresti fare un file `ApiConstants.java` in cui metterle